ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -184,15 +184,14 @@ StackLayout {
     }
 
     function clickOnNotification() {
+        // So far we're just showing this app as the top most window. Once we decide about the way
+        // how to notify the app what channle should be displayed within the app when user clicks
+        // notificaiton bubble this part should be updated accordingly.
+        //
+        // I removed part of this function which caused app crash.
         applicationWindow.show()
         applicationWindow.raise()
         applicationWindow.requestActivate()
-        appMain.changeAppSection(Constants.chat)
```

Comment:
I think maybe we can keep the `changeAppSection`?

Reply:
@jrainville to what? I was thinking also about that, but user may click on some bubble which belongs to one of channels of one of communities he has and we point him to the public chat section, so that's why I gave up from that idea. If we cannot determine the correct place where we should point him, why point him to the public chat section if he has some other section opened in that moment.

Reply:
Ah ok, let's wait until we improve the feature first then